### PR TITLE
solution

### PR DESCRIPTION
**Pull request by nazarko94**

### CODE REVIEW

File: src/transformStateWithClones.js [hunk 1]
```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = { ...state };
```
Comment (bohdan-stepanets) [inline src/transformStateWithClones.js:10]: Use relevant variable name

File: src/transformStateWithClones.js [hunk 2]
```diff
@@ -7,7 +7,34 @@
...
+  const copyArray = [];
```
Comment (bohdan-stepanets) [inline src/transformStateWithClones.js:11]: and here
point 5 of checklist

File: src/transformStateWithClones.js [hunk 3]
```diff
@@ -7,7 +7,34 @@
...
+      case 'removeProperties':
+        for (const props of action.keysToRemove) {
+          delete copy[props];
+        };
+        break;
+
+      case 'clear':
+        for (const key in copy) {
+          delete copy[key];
+        }
```
Comment (bohdan-stepanets) [inline src/transformStateWithClones.js:28]: Put blank line after cycle for better reading (not sure)

**Review by bohdan-stepanets — CHANGES_REQUESTED** [review]

**Review by bohdan-stepanets — CHANGES_REQUESTED** [review]
GJ 
Read checklist and fix mistakes

**Review by bohdan-stepanets — APPROVED** [review]
GJ 🔥

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**